daemon/mgr/system.go

```diff
@@ -96,8 +99,13 @@ func (mgr *SystemManager) Info() (types.SystemInfo, error) {
 		OSName = osName
 	}
 
+	images, err := mgr.imageMgr.ListImages(context.Background(), "")
+	if err != nil {
+		logrus.Warnf("failed to get image info")
```

Comment:
this error log seems improper. You did not describe the error correctly, and did not log the detailed error message, right?

How about `logrus.Warnf("failed to list images: %v", err)`? @ZouRui89

Reply:
The filed 'image' corresponds to the number of images, in which case I think ```logrus.Warnf("failed to get image info: %v", err)``` is better. WDYT? @allencloud